.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "seneca"
+  "extends": "seneca",
+  "rules": {
```

Comment:
Doing so, I would have replace the snake case by pascal one.

Because I have to say, `Auth_token` and *so*, feels a bit bit bit weird